app/routes/datasets/queries.py

```diff
@@ -95,9 +98,10 @@ async def query_dataset(
     else:
         geometry = None
 
-    data: List[RowProxy] = await _query_dataset(dataset, version, sql, geometry)
+    data: List[Dict[str, Any]]= await _query_dataset(dataset, version, sql, geometry)
+    serialized_data = orjson.dumps(data, default=jsonencoder_lite())
 
-    return Response(data=data)
+    return Response(content=serialized_data, media_type="application/json")
```

Comment:
Which `Response` class do you mean to reference here?
Right now you have `from ...models.pydantic.responses import Response` in scop. This is one of our pydantic models which assures that our responses follow the JSEND pattern.

It seems like you want to return a standard FastAPI `Response` which is not in scope. If doing so, please make sure you place the query results under the `data` key and add another root level key `"status":"success"`